doc/changelog/04-tactics/18591-master+simpl-refold-in-destructor-arg-position.rst

```diff
@@ -0,0 +1,5 @@
+- **Fixed:**
+  Refolding of constants marked as :g:`simpl never` in position of
+  argument of a destructor in :tacn:`simpl`
+  (`#18591 <https://github.com/coq/coq/pull/18591>`_,
+  by Hugo Herbelin).
```

Comment:
Could you more specific about the incorrect behavior that you fixed?  e.g. they should not have been refolded or ???

I suppose there's no bug report to refer to.  Hint: it's not too late to create one

Reply:
OK, done. And message extended.